node/README.md

```diff
@@ -1,204 +1,171 @@
-# Offchain Aggregator POC
+# Off-chain Aggregator
 
-## Further works to be done
+## Introduction
 
-- functionality
-- [ ] peer decision among pubsub subscribers (elect reporter)
+Offchain aggregator.
+It performs following steps to regularly submit data into chain
 
-- optimization
-- [ ] faster peer search
-- [ ] optimize pubsub peerfilter
-- [ ] check following options (libp2p host option, pubsub.option, discovery.option, pubsub.TopicOpt, pubsub.SubOpt)
-- [ ] is there a better way to generate id based on timerange? 🤔
+1. Fetch price data and save into database
+2. Send and Receive data with other nodes and aggregate all received data. Save aggregated data into database
+3. Submit aggregated data into chain
 
-- db integration
-- [ ] store data into db (redis or pgsql)
+![Overview](./image.svg)
 
-- migrate codes
-- [ ] fetcher codes
-- [ ] reporter codes: how to deal with caver.js 🤔
+- `submitter` is `reporter`.
```

Comment:
@nick-bisonai I think the terminology was change to `reporter` 🤔

Reply:
I'll make an update